app/views/cookies/show.html.erb

```diff
@@ -17,10 +17,10 @@
 
     <h1 class="govuk-heading-l">Cookies</h1>
     <p>Cookies are small files saved on your phone, tablet or computer when you visit a website.</p>
-    <p>We use cookies to make the Early Career Framework (ECF) work and collect information about how you use our service.</p>
+    <p>We use cookies to make the Support for early career teachers work and collect information about how you use our service.</p>
```

Comment:
in the other repo they've just written "the service" everywhere but the first time - might be easier to do the same here